axelor-account/src/main/resources/views/Move.xml

```diff
@@ -215,7 +215,7 @@
           readonlyIf="$validatePeriod" onClick="action-account-move-group-generate-tax-lines"
           colSpan="12" showIf="statusSelect == 1 || statusSelect == 5"/>
         <button
-          showIf="journal != null &amp;&amp; statusSelect != 3 &amp;&amp; ($journalTechnicalTypeSelect == 1 || $journalTechnicalTypeSelect == 2 || $journalTechnicalTypeSelect == 4)"
+          showIf="journal != null &amp;&amp; statusSelect != 3 &amp;&amp; (($journalTechnicalTypeSelect == 1 || $journalTechnicalTypeSelect == 2 || $journalTechnicalTypeSelect == 4) &amp;&amp; functionalOriginSelect != null &amp;&amp; (functionalOriginSelect == 3 || functionalOriginSelect == 4 || functionalOriginSelect == 7))"
```

Comment:
Please use an array with `include` instead of multiple equality checks. Also use it for existing `$journalTechnicalTypeSelect` checks. Also, `functionalOriginSelect != null` can be simplified to `functionalOriginSelect`

Reply:
Resolved